File: c/exercise11/currency_conversion.c
#include <stdio.h>
#include <stdlib.h>

int main(void) {
  float euros = 0.0;
  printf("How many euros are you exchanging? ");
  scanf("%f", &euros);

  float exchange_rate = 0.0;
  printf("What is the exchange rate? ");
  scanf("%f", &exchange_rate);

  float to_dollars = (euros * exchange_rate) / 100;
  printf("%.2f euros at an exchange rate of %.2f is\n%.2f U.S. dollars.\n",
         euros, exchange_rate, to_dollars);

  return EXIT_SUCCESS;
}

File: javascript/currency-conversion/currency-conversion.js
import { input } from "pynput";

const euros = Number(await input("How many euros are you exchanging? "));
const exchange_rate = Number(await input("What is the exchange rate? "));

const to_dollars = (euros.toFixed(2) * exchange_rate) / 100;
console.log(
	`${euros} euros at an exchange rate of ${exchange_rate} is\n${to_dollars.toFixed(2)} U.S. dollars.`,
);
